package.json

```diff
@@ -4,7 +4,8 @@
   "author": "Tobias Koppers @sokra",
   "description": "less loader module for webpack",
   "scripts": {
-    "test": "node --no-deprecation node_modules/.bin/_mocha -R spec",
+    "test": "npm run travis:test",
+    "travis:test": "node --no-deprecation node_modules/.bin/_mocha -R spec",
```

Comment:
`Why is there an extra command for travis:test? We should just define test`

Look at the .travis.yml. The matrix executes different `parts` across different supported node versions. A part is defined as `travis:<part>`

`I know that this has been there before, but why --no-deprecation`

I have no idea, left it alone mainly becuase I don't know why it was there in the first place.

I can either remove it now or it will be removed on the defaults upgrade to Jest. Your choice.